src/main/java/pw/itr0/kaba/encrypt/PBE.java

```diff
@@ -189,16 +189,13 @@ private Encrypter(byte[] salt, int iterationCount) {
         }
 
         /**
-         * Encrypt secret bytes using password for PBE.
-         *
-         * @param password encrypting password
-         * @param secret   secret bytes
-         * @return PBE encrypted secret
-         * @throws GeneralSecurityException on encryption error
+         * {@inheritDoc}
+         * <p>
+         * This implementation return PBE encryption {@link Cipher}.
          */
         @Override
-        public byte[] encrypt(char[] password, byte[] secret) throws GeneralSecurityException {
-            return getCipher(Cipher.ENCRYPT_MODE, password, this.salt, this.iterationCount).doFinal(secret);
+        public Cipher getEncryptionCipher(char[] password) throws InvalidKeyException {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/ryotan/kaba-vault/file/2671050498/issues/source?bid=3298589&fileBranchId=3332567#l197)